handbook/tools/email.md

```diff
@@ -0,0 +1,26 @@
+# ✉️ Email
```

Comment:
everything hear is currently active (the strategy account, automatic forwarding, etc)